build/args/release.gn

```diff
@@ -0,0 +1,7 @@
+import("all.gn")
+is_component_build = false
+is_official_build = true
+enable_widevine = true
+proprietary_codecs = true
+is_cfi = false # TODO(nornagon): why disable this?
```

Comment:
I am building with cfi enabled in the release build. Since node is no longer a shared lib, it should be fine enabling it.